commonskmm/src/commonMain/kotlin/org/dhis2/mobile/commons/model/CustomIntentModel.kt

```diff
@@ -27,3 +27,8 @@ enum class CustomIntentResponseExtraType {
     OBJECT,
     LIST_OF_OBJECTS,
 }
+
+enum class CustomIntentActionTypeModel {
```

Comment:
Not to be changed, but I think we should be using sealed interfaces with data objects instead of enums. Maybe in the future we want to expand the SEARCH with a set of parameters or something like that and having data classes/objets are easier to update. Also, for multiplatform I think enums could be problematic.